### ISSUE TITLE
How to continue tagging on an already tagged image

### ISSUE THREAD

**Issue by liangpu597** [tzutalin/labelImg#625]
- **OS:** Ubuntu16.04
- **PyQt version:**1.8.1
I have marking "human" label  in all of my train image. and i have train them . and finish. 
now i want to tagging more label in original picture 。 for example i want to tag "car ", "motor" in origin picture and keep original "human" label stay,
please tell me how to do ?  thank you!

**Comment by AnhPC03** [comment]
@liangpu597 Yes, you can. Open labelImg then "open dir" to open folder which contains your images. In left bar, choose "change save dir" and then open folder which you saved "human" label to.
